### PR TITLE
Translate design systems for developers to zh-CN

### PR DESCRIPTION
**Pull request by XiKun-TW**
Proofreading from native speakers wanted. Review and comment on this PR 🙏.

### CODE REVIEW

File: content/design-systems-for-developers/react/zh-CN/document.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/document.md]:
```suggestion
作为 Storybook 的用户，我们具有领先优势，因为组件的各个变体已经被记录为 stories（一种文档形式）。一个 story 展示了组件在不同输入下应该如何工作， story 利于编写，且因其引用了生产环境的组件，所以可以自动更新。并且，我们可以使用上一章中的工具对 story 进行回归测试！
```

File: content/design-systems-for-developers/react/zh-CN/distribute.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/distribute.md]:
```suggestion
赞! 我们已经成功的在 npm 中发布了我们的软件包并且在 GitHub 上创建了一个 release（祝好运!）。
```

File: content/design-systems-for-developers/react/zh-CN/distribute.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/distribute.md]:
```suggestion
将您发布的设计系统添加为依赖项：
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
感谢 Storybook Docs 插件，我们只需要花一点点时间便可以得到内容可自定义的文档。它可以帮助其他人通过浏览 Storybook 的 Docs 页签去学习如何使用 AvatarList。
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
因为 AvatarList 是一个新的组件且未被视觉测试覆盖到，我们则需要为每个 story 去添加新的基准。在 Chromatic 中接受 “新 stories” 去扩大我们的视觉测试覆盖率。
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
测试被成功的更新了。在以后，回溯将很难偷偷摸摸地就被加入到设计系统中。
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
我们有一个进行的 pull request 去把 AvatarList 添加到设计系统中。我们已经编写了相关的 stories，测试已经通过并且也有相关的文档。最后，我们准备好使用 Auto 和 npm 来打包设计系统。
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
现在合并您的 pull request 到主分支，您可以在 npm 网站中打开您的软件包并稍等几分钟去查看新的版本。
```

File: content/design-systems-for-developers/react/zh-CN/workflow.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/workflow.md]:
```suggestion
_针对开发人员的设计系统_ 着重介绍了专业团队使用的端到端的工作流程，帮助您能在开发自己的设计系统时先行一步。伴随着您的设计系统的发展、整理，您可以扩展这些工具来满足您团队的需求。
```

File: content/design-systems-for-developers/react/zh-CN/test.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/test.md]:
```suggestion
**清楚地将组件支持的状态**表达为 stories 可以阐明不同的输入组合所对应的组件状态，尽可能去忽略不支持的状态来降低干扰。
```

File: content/design-systems-for-developers/react/zh-CN/test.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/test.md]:
```suggestion
之前我们已经配置了我们的 Circle config.js 在每一次提交之后运行 `yarn test`。现在我们这条单元测试便可从中获益。在之后的反复修改组件时，我们也会对强大的 Link 组件保持信心。
```

File: content/design-systems-for-developers/react/zh-CN/introduction.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/introduction.md]:
```suggestion
在设计系统中，文档是必不可少的，但是开发人员往往将创建文档作为最低的优先级。我们通过 UI 组件将会为您提供自动生成最小可行的文档，极大地降低了您开发文档的难度，您也可以自行修改自动生成的文档。
```

File: content/design-systems-for-developers/react/zh-CN/introduction.md
Comment (plrthink) [inline content/design-systems-for-developers/react/zh-CN/introduction.md]:
```suggestion
设计系统既是唯一的“真理之源”，也是 UI 组件唯一的 “错误之源”。设计系统中一个小小的 UI 错误，可能会像滚雪球一样导致整个产品出现问题。我们将自动执行测试来帮助您规避不可避免的错误，并让您能放心地交付稳定、无障碍访问的 UI 组件。
```

File: content/design-systems-for-developers/react/zh-CN/document.md
Comment (XiKun-TW) [reply content/design-systems-for-developers/react/zh-CN/document.md]: The en version is `for free` I think it's better to translate it to `免费`. It also make sense on current context.

**Comment by XiKun-TW** [comment]
Hi @plrthink , I made some changes based on your suggestions. Except `for free` (open for others opinions), the others are very helpful. Thanks !

**Comment by plrthink** [comment]
My pleasure. I'm happy to get involved.

File: content/design-systems-for-developers/react/zh-CN/document.md
Comment (plrthink) [reply content/design-systems-for-developers/react/zh-CN/document.md]: Well, I think '免费的' emphasizes on the finance and the words want to express the low effort on generating the docs, like the word 'bonus'.

But I'm open on this.

File: content/design-systems-for-developers/react/zh-CN/introduction.md
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/introduction.md]: 这些 UI 组件经过重新排列来展现出不同的用户体验。
->这些 UI 组件经过重新排列来提供不同的用户体验。

File: content/design-systems-for-developers/react/zh-CN/introduction.md
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/introduction.md]: 有大量面向设计师的指南，深入探讨了这些主题，因此我们不在这里赘述。
->有大量面向设计师的指南已经深入探讨过这些主题，因此我们不在这里赘述。

File: content/design-systems-for-developers/react/zh-CN/introduction.md
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/introduction.md]: 如果您发现您自己复制粘贴了相同的 UI 组件在不同的应用程序或者团队之间
->如果您发现您自己在不同的应用程序或者团队之间复制粘贴了相同的 UI 组件

**Comment by domyen** [comment]
🙏 Thank you for contributing @plrthink @PhonyLou @XiKun-TW! I'm super excited for this.

**Comment by XiKun-TW** [comment]
Hi @PhonyLou , Thanks for your comment. It's very helpful.

File: content/design-systems-for-developers/react/zh-CN/conclusion.md
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/conclusion.md]: 不，这可能会引发创建和维护的成本。在规模的设计生产中，一个设计系统节省的时间可能会小于创建它而耗费的时间。
->
不，这会带来开发和维护上的机会成本。在小规模应用上，使用设计系统所需的工作量代价比节省出来的时间更大。

File: content/design-systems-for-developers/react/zh-CN/introduction.md
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/introduction.md:61]:
```suggestion
- ✅ [Chromatic](https://www.chromatic.com/) 用于捕获组件中的视觉错误 (由 Storybook 维护人员执行)
```

File: content/design-systems-for-developers/react/zh-CN/test.md
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:42]:
```suggestion
首先，通过您的 Github 在 [Chromatic.com](https://www.chromatic.com/) 上注册一个账号
```

File: content/design-systems-for-developers/react/zh-CN/test.md
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:50]:
```suggestion
通过 npm 安装 [chromatic](https://www.npmjs.com/package/chromatic)
```

File: content/design-systems-for-developers/react/zh-CN/test.md
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:53]:
```suggestion
yarn add --dev chromatic
```

File: content/design-systems-for-developers/react/zh-CN/test.md
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:59]:
```suggestion
npx chromatic --project-token=<project-token>
```

File: content/design-systems-for-developers/react/zh-CN/test.md
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:81]:
```suggestion
npx chromatic --project-token=<project-token>
```

File: content/design-systems-for-developers/react/zh-CN/test.md
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/test.md:117]:
```suggestion
      - run: yarn chromatic --project-token=<project-token> --exit-zero-on-changes
```

File: content/design-systems-for-developers/react/zh-CN/distribute.md
Comment (ghengeveld) [inline content/design-systems-for-developers/react/zh-CN/distribute.md:249]:
```suggestion
- run: yarn chromatic --project-token=2wix88i1ziu
```

**Comment by XiKun-TW** [comment]
Thanks @ghengeveld, I already accept the all commits.

**Comment by jonniebigodes** [comment]
@plrthink @PhonyLou @XiKun-TW this is a friendly reminder to see how this is going? Also i don't know if you've checked but the design systems tutorial was updated recently. 

If you're able can you make the changes so that the tutorial is updated to the latest version and we get a accurate translation?

Stay safe all of you

**Comment by PhonyLou** [comment]
Hey @jonniebigodes , I cannot make the change merged. Maybe @XiKun-TW can help.

Hope you guys all good and safe~

**Comment by XiKun-TW** [comment]
Hi @jonniebigodes , I'll check all the changes, thanks for reminding. :-D

**Comment by jonniebigodes** [comment]
@XiKun-TW no need to thank. Glad that this will get updated soon and we'll get it merged once it's all fine tuned. Sorry for not reminding you earlier, but these have been a bit of a hectic weeks. 

@PhonyLou once @XiKun-TW addresses the feedback i would like for you to take another look and proofread it, so that we can get it merged.

And it goes without saying thank you very much for taking the time to translate this

Stay safe both of you.

**Comment by XiKun-TW** [comment]
Hi @jonniebigodes @PhonyLou , the latest translate is ready for proofread again. Pls help review, Thanks.

**Comment by plrthink** [comment]
I just reviewed the changes made these days, and I think it looks good to me.

**Comment by jonniebigodes** [comment]
@XiKun-TW i'll take a look shortly and see if we're good to go, thank you @plrthink for taking the time to take a look at the translation as well. 

I'll let you know before the merge is done.

File: content/design-systems-for-developers/react/zh-CN/review.md
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/review.md:75]: 这便可以让您的团队非常轻而易举的能够看到和您本地一模一样的 UI 组件。
->
这便可以让您的团队轻而易举地看到和您本地一模一样的 UI 组件。

File: content/design-systems-for-developers/react/zh-CN/review.md
Comment (PhonyLou) [inline content/design-systems-for-developers/react/zh-CN/review.md:128]: 最后通过如下命令将您对提交上传到远端代码仓库:
->
最后通过如下命令将您的提交上传到远端代码仓库：

**Comment by PhonyLou** [comment]
For others, I think they are good enough. Thanks.

**Comment by XiKun-TW** [comment]
> For others, I think they are good enough. Thanks.

Thanks for review! @jonniebigodes, Could you pls help confirm wether if it's ok to merge to master?

**Comment by jonniebigodes** [comment]
@XiKun-TW sorry for not addressing this earlier, but it's been some hectic months with the release of Storybook 6.0 and updating the tutorials.

I wanted to know if you're ok with updating this to the most recent version of the tutorial, now that #380 was merged? I was thinking that as groundwork is already done, it would be a waste to merge this now and if you wanted to make a follow up pull request with the new updates.

Let me know if you're ok with updating the tutorial so that we can work from there and offer the best/most recent translation to the community.

Stay safe.

**Comment by XiKun-TW** [comment]
> @XiKun-TW sorry for not addressing this earlier, but it's been some hectic months with the release of Storybook 6.0 and updating the tutorials.
> 
> I wanted to know if you're ok with updating this to the most recent version of the tutorial, now that #380 was merged? I was thinking that as groundwork is already done, it would be a waste to merge this now and if you wanted to make a follow up pull request with the new updates.
> 
> Let me know if you're ok with updating the tutorial so that we can work from there and offer the best/most recent translation to the community.
> 
> Stay safe.

Sure, I'll update translate to recent version.

File: content/design-systems-for-developers/react/zh-CN/document.md
Comment (MichaelArestad) [inline content/design-systems-for-developers/react/zh-CN/document.md:95]: Hey! I just noticed this reference to a `components` directory and created [a PR to remove it](https://github.com/chromaui/learnstorybook.com/pull/508). I would remove it here and a handful of times where it appears in code further down in this document.

**Review by ghengeveld — APPROVED** [review]
Time to get this thing merged.

**Pull request merged**
